bindings/nodejs/lib/wallet/wallet.ts

```diff
@@ -1315,6 +1317,147 @@ export class Wallet {
         );
     }
 
+    /**
+     * Begin staking.
+     *
+     * @param params The options for begging a staking.
+     * @param transactionOptions Additional transaction options or custom inputs.
```

Comment:
```suggestion
     * @param transactionOptions Additional transaction options.
```
If ever you feel like doing this in the whole file, it's a bit stupid to mention custom inputs as they are in the options. Feel free to resolve and we do it in another PR.

Reply:
done

Reply:
thanks